client/src/features/session/sessions.utils.ts

```diff
@@ -38,7 +38,9 @@ export function getRunningSession({
       namespace: annotations.namespace,
       path: annotations.projectName,
     });
-    return thisAutostartUrl === autostartUrl;
+    return typeof thisAutostartUrl !== "string"
+      ? thisAutostartUrl === autostartUrl
+      : thisAutostartUrl.toLowerCase() === autostartUrl.toLowerCase();
```

Comment:
This is always a string:

```suggestion
    return thisAutostartUrl.toLowerCase() === autostartUrl.toLowerCase();
```

In [client/src/utils/helpers/url/Url.js - L260](https://github.com/SwissDataScienceCenter/renku-ui/blob/66c40019326a8f66aa687d69cbb1f83617c8d939/client/src/utils/helpers/url/Url.js#L260), update the types:

```js
/**
 * Create a Url based on the target page. Depending on the specific page, it may require context data.
 *
 * @param {object | string} target - the page you are targeting, as contained in the `pages`  member
 *   (e.g. pages.landing, pages.project.stats, ...).
 * @param {object} [data] - the context data you need to provide, if any
 *   (e.g. for project, you need to provide a `namespace` and a `path`).
 * @param {boolean} [full] - switch between full or relative path. The default is `false`.
 * @returns {string}
 */
function get(target, data, full = false) {
```